cluster-autoscaler/utils/drain/drain_test.go

```diff
@@ -461,6 +496,24 @@ func TestDrain(t *testing.T) {
 			expectPods:          []*apiv1.Pod{failedPod},
 			expectDaemonSetPods: []*apiv1.Pod{},
 		},
+		{
+			description: "pod in unknown state - skipTerminatingPods",
```

Comment:
Could you make the description more accurate? E.g. something like "long terminating pod with 0 grace period".

Reply:
done